CoffeeChat/SceneDelegate.swift

```diff
@@ -6,18 +6,31 @@
 //  Copyright © 2020 cuappdev. All rights reserved.
 //
 
+import GoogleSignIn
 import UIKit
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
     var window: UIWindow?
 
-
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
         // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
-        guard let _ = (scene as? UIWindowScene) else { return }
+        guard let scene = scene as? UIWindowScene else { return }
+
+        let window = UIWindow(windowScene: scene)
+        let loginVC = LoginViewController()
+        let homeVC = HomeViewController()
```

Comment:
we can just use these values directly instead of storing it in a constant b/c we only use it once anyways